frontend/src/app/store/selectors/auth-page.selectors.ts

```diff
@@ -13,7 +13,7 @@ export const selectUser = createSelector(
 
 export const selectToken = createSelector(
     selectAuthState,
-    (state) => state.token
+    (state) => state.token ?? {} as Token
```

Comment:
Try avoid creating objects like this. Better use constructors. There are some important benefits that may keep you from errors in future. https://blog.logrocket.com/writing-constructor-typescript/

Reply:
Fixed by using helper for creating default instance